plugins/hydra_optuna_sweeper/setup.py

```diff
@@ -26,7 +26,10 @@ def get_long_description() -> str:
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
-        "Operating System :: OS Independent",
+        # "Programming Language :: Python :: 3.9",
+        # Python 3.9 support depends on https://github.com/optuna/optuna/issues/2034
+        "Operating System :: POSIX :: Linux",
+        "Operating System :: MacOS",
```

Comment:
why was windows removed here?

Reply:
ahh my bad. Thanks for catching!

Reply:
Okay, now I recall the problem was this warning `DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.9 it will stop working` on windows + Python 3.7 and 3.8

The error did not surface earlier because testing on windows (for 3.8 and 3.7) was disabled.

Reply:
The tests are run with `pytest -Werror tests` and ignoring warnings via pytest still cause the error. Will take another short today.

Reply:
Relevant CI run https://app.circleci.com/pipelines/github/facebookresearch/hydra/5418/workflows/52f8552a-3e49-4909-b2a5-8a2d4cb99bed/jobs/39570

Reply:
Since Optuna wasnt tested on windows earlier, I think we should track this in a separate issue. cc @omry

Reply:
I agree.

Reply:
Done. New issue: https://github.com/facebookresearch/hydra/issues/1218. All tests pass now